static/js/main.js

```diff
@@ -37,13 +37,24 @@ photosRef.once("value")
   });
 
 $('.submit').click(function () {
-  firebase.database().ref('photos/' + $('#title').val()).set({
-    title: $('#title').val(),
-    url: $('#url').val(),
-    location: $('input[name="location"]:checked').val(),
-    tags: $('#tags').val(),
-    likes: 0
-  });
+   var url = $('#url').val();
+
+          if(url.match(/(jpg|png)$/)){
```

Comment:
@Bouncey you should allow `gif` here also. Please update the alert message as well.